vcbuild.bat

```diff
@@ -49,7 +48,6 @@ if /i "%1"=="clean"         set target=Clean&goto arg-ok
 if /i "%1"=="ia32"          set target_arch=x86&goto arg-ok
 if /i "%1"=="x86"           set target_arch=x86&goto arg-ok
 if /i "%1"=="x64"           set target_arch=x64&goto arg-ok
-if /i "%1"=="vc2015"        set target_env=vc2015&goto arg-ok
```

Comment:
Ok.
It has been a blind switch since [build: remove VS 2013 switch from vcbuild.bat](https://github.com/nodejs/node/commit/a3f861de9d1ab43934fa8de6352a9a98ddefd44a) but now it makes sense again...